pyrasterframes/src/main/python/setup.py

```diff
@@ -72,6 +69,59 @@ def run(self):
                 print(traceback.format_exc())
 
 
+class PweaveDocs(distutils.cmd.Command):
+    """A custom command to run documentation scripts through pweave."""
+    description = 'Pweave PyRasterFrames documentation scripts'
+    user_options = [
+        # The format is (long option, short option, description).
+        ('files=', 'f', 'Specific files to pweave. Defaults to all in `docs` directory.'),
+    ]
+
+    def initialize_options(self):
+        """Set default values for options."""
+        # Each user option must be listed here with their default value.
+        self.files = filter(
+            lambda x: not x.name.startswith('_'),
+            glob(path.join(here, 'docs', '*.py'))
+        )
+
+    def finalize_options(self):
+        """Post-process options."""
+        import re
+        if isinstance(self.files, str):
+            self.files = filter(lambda s: len(s) > 0, re.split('\W+', self.files))
+
+    def run(self):
+        """Run pweave."""
+        import traceback
+        import pweave
+
+        for file in self.files:
+            name = path.splitext(path.basename(file))[0]
+            print(_divided('Running %s' % name))
+            try:
+                pweave.weave(
+                    file=str(file),
+                    doctype='markdown'
+                )
+            except Exception:
+                print(_divided('%s Failed:' % file))
+                print(traceback.format_exc())
+
+
+try:
+    with open(path.join(here, 'requirements.txt')) as f:
+        requirements = f.read().splitlines()
+except:
```

Comment:
I think this is in a bit of an odd state. I believe if you add requirements.txt to the MANIFEST this is not necessary.

I had previously had this fail in the pip installed package because there was no file at $(here)/requirements.txt